src/Library/demos/Drag and Drop/main.js

```diff
@@ -0,0 +1,67 @@
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+
+const list = workbench.builder.get_object("list");
+
+// Iterate over ListBox children
+for (const row of list) {
+  row.selectable = false;
+  let _drag_x;
+  let _drag_y;
+
+  const drag_source = new Gtk.DragSource({
+    actions: Gdk.DragAction.MOVE,
+  });
+
+  const drop_target = Gtk.DropTarget.new(
+    row.constructor.$gtype,
```

Comment:
This returns the correct `GType`, as confirmed by logging, but for some reason the drop never triggers unless it's `Gtk.ListBoxRow`. I'm not sure why that is, but if it works for you too, lets go for that.

Reply:
We can change it to Gtk.ListBoxRow. That is how I read I should get types, but it doesn't work for me either.